run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Run database installation only once
+./listmonk --install --idempotent --yes
```

Comment:
This is incorrect. `--config=''`, `--upgrade=''` etc. are necessary for listmonk to auto-updated on boot when the image version changes. Why were they removed?